project.clj

```diff
@@ -59,7 +59,7 @@
                  [metosin/schema-tools "0.12.2"]
                  [threatgrid/flanders "0.1.23"]
 
-                 [threatgrid/ctim "1.0.23"]
+                 [threatgrid/ctim "1.0.24-SNAPSHOT"]
```

Comment:
need to deploy CTIM release after merging this PR: https://github.com/threatgrid/ctim/pull/320